devops/deploy/spin-up-script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PSTART=${1:-50000}
+PEND=${2:-50050}
+# Just a basic script to spin up listening servers on different ports in dynamic port range
+IP=127.0.0.1
+COUNT=0
+((NUM_PORTS=${PEND}-${PSTART}))
+
+echo "Total number of ports: $NUM_PORTS"
+export PYTHONPATH=../../src/main/py
+
+port=$PSTART
+
+while [ ${COUNT} -lt ${NUM_PORTS} ]
+do
+  #Check for Open ports, if open then OK, else continue
+  if ! netstat -apn | grep -q "$IP:$port" 
```

Comment:
Similarly, `lsof` will return an non `0` result if the port isn't bound, so is available. In this example, the first port is bound so it returns a `0` result and prints the PID of the process bound because of `-t`. The second isn't bound so a non-`0` result is returned and nothing is printed.

```
2024-01-18 16:08:56 ~
1012 | 0 $ lsof -ti :42422
24877
2024-01-18 16:09:06 ~
1013 | 0 $ lsof -ti :42423
2024-01-18 16:09:09 ~
1014 | 1 $
```